pyworkflow/pyworkflow/workflow.py

```diff
@@ -19,13 +16,16 @@ class Workflow:
         file_path: Location of a workflow file
     """
 
-    def __init__(self, graph=nx.DiGraph(), file_path=None, name='a-name', flow_vars=nx.Graph(), file_system=fs):
+    def __init__(self, graph=nx.DiGraph(), file_path=None, name='a-name', flow_vars=nx.Graph(), root_dir=settings.MEDIA_ROOT):
```

Comment:
I tried writing a pyworkflow unit test that imported `django.conf.settings`, and it complains about Django not being configured. So I would either set `/tmp` as the default, or make it `None` and then set it to `os.getcwd()` when checking its existence in `__init__`. That would allow someone to use pyworkflow as a regular-old-python-module and have all their output where they're working.